com-api/src/lib.rs

```diff
@@ -40,27 +40,32 @@
 //! - Structures
 //! - Tuples
 
+use std::collections::VecDeque;
 use std::fmt::Debug;
-use std::mem::MaybeUninit;
 use std::ops::{Deref, DerefMut};
 use std::path::Path;
 
 #[derive(Debug)]
 pub enum Error {
+    /// TODO: To be replaced, dummy value for "something went wrong"
     Fail,
+    Timeout,
 }
```

Comment:
I'm not sure how to formulate an issue out of this ... could you @darkwisebear  do it? Or is that one of the todos we would like to keep in the code directly?

Reply:
I think it's ok to not have an issue in that case. `Fail` is really only a placeholder and when we gather more and more error categories, this one can go away eventually in favor of more precise errors.